
Traceback (most recent call last):
  File "tests/exceptions/source/others/catch_as_decorator_with_parentheses.py", line 14, in <module>
    c(5, b=0)
  File "tests/exceptions/source/others/catch_as_decorator_with_parentheses.py", line 11, in c
    a / b
ZeroDivisionError: division by zero
